packages/go_router/test/flutter_test_config.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:async';
+
+import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
+
+FutureOr<void> testExecutable(FutureOr<void> Function() testMain) {
+  LeakTesting.enable();
+  LeakTesting.settings =
+      LeakTesting.settings.withIgnored(createdByTestHelpers: true);
```

Comment:
any reason we are ignoring this?

Reply:
This is was https://github.com/dart-lang/leak_tracker/blob/main/doc%2Fleak_tracking%2FDETECT.md suggests to do.
Maybe @polina-c would have more insights

Reply:
If test helpers create a disposable and then do not dispose it, it is not a big deal as it does not affect production code.
This setting helps to setup leak tracking faster for existing projects where test helpers are leaking.
But, it is up to each project if this should set.
Does it help?

Reply:
Thank you @polina-c 

@chunhtai [test: Don't ignore leaks created by test helpers](https://github.com/flutter/packages/pull/7546/commits/f4912f7cc083418ac7901939e3622eb63ae51f7d) removes the ignore